src/printer.rs

```diff
@@ -11,7 +11,7 @@ pub struct Printer {
 }
 
 impl Visitor for Printer {
-    fn visit(&self, tree: &Tree, path: &[(usize, usize)]) {
+    fn visit(&self, tree: &Tree, path: &[(usize, usize)], _: &[&str]) {
```

Comment:
Better to use `_parents:` instead of just `_:` here. When I was skimming through this part I was confused by what this parameter is and why it was being omitted.